README.md

```diff
@@ -1,5 +1,5 @@
 # nvOC easy-to-use Linux Nvidia Mining OS
-Looking for Latest stable version? Go here: https://github.com/papampi/nvOC_by_fullzero_Community_Release/tree/19.2
+This page is about the Latest stable version of nvOC Community Edition
```

Comment:
Revert this line change, use something similar to the old beta alert but pointing to 3.0 instead.